test/signers/pss_signer_test.dart

```diff
@@ -0,0 +1,200 @@
+    BigInt.parse('010001', radix: 16),
+    BigInt.parse(
+        '6a7fd84fb85fad073b34406db74f8d61a6abc12196a961dd79565e9da6e5187bce2d980250f7359575359270d91590bb0e427c71460b55d51410b191bcf309fea131a92c8e702738fa719f1e0041f52e40e91f229f4d96a1e6f172e15596b4510a6daec26105f2bebc53316b87bdf21311666070e8dfee69d52c71a976caae79c72b68d28580dc686d9f5129d225f82b3d615513a882b3db91416b48ce08888213e37eeb9af800d81cab328ce420689903c00c7b5fd31b75503a6d419684d629',
+        radix: 16),
+    BigInt.parse(
+        'f8eb97e98df12664eefdb761596a69ddcd0e76daece6ed4bf5a1b50ac086f7928a4d2f8726a77e515b74da41988f220b1cc87aa1fc810ce99a82f2d1ce821edced794c6941f42c7a1a0b8c4d28c75ec60b652279f6154a762aed165d47dee367',
+        radix: 16),
+  );
+
+  // PSS Example 9.1
+  var msg9a = hex.decode(
+      'a88e265855e9d7ca36c68795f0b31b591cd6587c71d060a0b3f7f3eaef43795922028bc2b6ad467cfc2d7f659c5385aa70ba3672cdde4cfe4970cc7904601b278872bf51321c4a972f3c95570f3445d4f57980e0f20df54846e6a52c668f1288c03f95006ea32f562d40d52af9feb32f0fa06db65b588a237b34e592d55cf979f903a642ef64d2ed542aa8c77dc1dd762f45a59303ed75e541ca271e2b60ca709e44fa0661131e8d5d4163fd8d398566ce26de8730e72f9cca737641c244159420637028df0a18079d6208ea8b4711a2c750f5');
+  var slt9a = hex.decode('c0a425313df8d7564bd2434d311523d5257eed80');
+  var sig9a = hex.decode(
+      '586107226c3ce013a7c8f04d1a6a2959bb4b8e205ba43a27b50f124111bc35ef589b039f5932187cb696d7d9a32c0c38300a5cdda4834b62d2eb240af33f79d13dfbf095bf599e0d9686948c1964747b67e89c9aba5cd85016236f566cc5802cb13ead51bc7ca6bef3b94dcbdbb1d570469771df0e00b1a8a06777472d2316279edae86474668d4e1efff95f1de61c6020da32ae92bbf16520fef3cf4d88f61121f24bbd9fe91b59caf1235b2a93ff81fc403addf4ebdea84934a9cdaf8e1a9e');
+
+  // PSS Example 9.2
+  var msg9b = hex.decode(
+      'c8c9c6af04acda414d227ef23e0820c3732c500dc87275e95b0d095413993c2658bc1d988581ba879c2d201f14cb88ced153a01969a7bf0a7be79c84c1486bc12b3fa6c59871b6827c8ce253ca5fefa8a8c690bf326e8e37cdb96d90a82ebab69f86350e1822e8bd536a2e');
+  var slt9b = hex.decode('b307c43b4850a8dac2f15f32e37839ef8c5c0e91');
+  var sig9b = hex.decode(
+      '80b6d643255209f0a456763897ac9ed259d459b49c2887e5882ecb4434cfd66dd7e1699375381e51cd7f554f2c271704b399d42b4be2540a0eca61951f55267f7c2878c122842dadb28b01bd5f8c025f7e228418a673c03d6bc0c736d0a29546bd67f786d9d692ccea778d71d98c2063b7a71092187a4d35af108111d83e83eae46c46aa34277e06044589903788f1d5e7cee25fb485e92949118814d6f2c3ee361489016f327fb5bc517eb50470bffa1afa5f4ce9aa0ce5b8ee19bf5501b958');*/
+
+  var pubParams =
+      (RSAPublicKey pubk, Uint8List salt) => () => ParametersWithSalt(
+            PublicKeyParameter<RSAPublicKey>(pubk),
+            salt,
+          );
+  var privParams = (RSAPrivateKey privk, Uint8List salt) =>
+      () => ParametersWithSalt(PrivateKeyParameter<RSAPrivateKey>(privk), salt);
+
+  var slt1A =
+      createUint8ListFromHexString('dee959c7e06411361420ff80185ed57f3e6776af');
+  runSignerTests(
+      Signer('SHA-1/PSS'), privParams(prv1, slt1A), pubParams(pub1, slt1A), [
```

Comment:
I tried porting the tests over but sadly it looks like the current test infra doesn't work well with them.

1) There's no way to pass a different salt for each test case and the salt would be shared for each instance of `Signer`.
2) `runSignerTests` internally uses `createUint8ListFromString` instead of `createUint8ListFromHexString`. (so these tests don't pass anymore)

I could create new test cases with the Java implementation to fit the testing model here better if that's what we want to go with.